prow/cluster/sinker_deployment.yaml

```diff
@@ -23,6 +23,12 @@ spec:
         - --config-path=/etc/config/config.yaml
         - --job-config-path=/etc/job-config
         - --dry-run=false
+        - --namespace=$(MY_POD_NAMESPACE)
+       env:
+       - name: MY_POD_NAMESPACE
+         valueFrom:
+          fieldRef:
+            fieldPath: metadata.namespace
```

Comment:
This isn't the `test-pods` namespace is it?

Reply:
Actually looks more like the env var name is misleading. Should be `MY_PROWJOB_NAMESPACE`?

We really should be consuming theses values from config though. We shouldn't assume that `sinker` is running in the same namespace the ProwJobs are in: https://github.com/kubernetes/test-infra/blob/036124a3a282bac8859218cf7112721f151b71c1/prow/config/config.go#L95-L104

Reply:
It's the namespace the Sinker Pod runs in, it's being used for the leader election only